source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidRealTimeMultiplayerClient.cs

```diff
@@ -11,16 +11,36 @@ namespace GooglePlayGames.Android
 
     internal class AndroidRealTimeMultiplayerClient : IRealTimeMultiplayerClient
     {
+        private readonly object mSessionLock = new object();
+
+        private const float InitialPercentComplete = 20.0F;
+
+        private static readonly int ROOM_VARIANT_DEFAULT = -1;
```

Comment:
>ROOM_VARIANT_DEFAULT

We should mention it should be synced with java constant

Reply:
I removed this variable as I realized it is for something else but not for room status in Java SDK.